src/dune/dune.py

```diff
@@ -130,14 +130,12 @@ def start_node(self, nod, snapshot=None):
         # copy config.ini to config-dir
         if not is_restart and nod.config() is None:
             current_ver = self.get_current_nodeos_version()
-            config_ini = get_config_ini(self._docker.get_arch(), *current_ver)
-            with tempfile.NamedTemporaryFile("w+", encoding="UTF-8", delete=False) as config_file :
-                config_file.write(config_ini)
-                config_file.close()
-                self._docker.cp_from_host(config_file.name, "/app/config.ini")
-                os.remove(config_file.name)
+            config_str = get_config_ini(self._docker.get_arch(), current_ver[0], current_ver[1], current_ver[2])
+            self._docker.write_file("/app/config.ini", config_str)
             nod.set_config('/app/config.ini')
 
+
+
```

Comment:
3 blank lines seems like 1 too mant? Nice to fix, not necessary to fix.

Reply:
Right, done